ext/RMagick/rmutil.c

```diff
@@ -688,7 +703,12 @@ rm_pixelpacket_to_color_name_info(Info *info, PixelPacket *color)
 
     my_info = info ? info : CloneImageInfo(NULL);
```

Comment:
(This is no relationship to PR)
I think we should raise an exception or assert if NULL was passed via `info` to notify coding miss.

All `rm_pixelpacket_to_color_name_info()` calling are paired with `Data_Get_Struct()`.
If `Data_Get_Struct()` returns `NULL` as `info` with `self` object, I think Ruby runtime has been broken...
https://github.com/rmagick/rmagick/blob/a828b4e36466ec87544211b5eee055270012c74e/ext/RMagick/rminfo.c#L491-L492